_maps/map_files/NorthStar/north_star.dmm

```diff
@@ -19198,6 +19292,7 @@
 	dir = 4
 	},
 /obj/structure/rack,
+/obj/item/storage/box/pinpointer_pairs,
```

Comment:
I don't think I've ever actually seen these in-game. Do they work? Interesting lil addition.

Reply:
> I don't think I've ever actually seen these in-game. Do they work? Interesting lil addition.

I used those for events before, they're neat.